pkg/openfeature/client.go

```diff
@@ -5,6 +5,10 @@ import (
 	"fmt"
 )
 
+const (
+	TargetingKey string = "targetingKey" // flattened evaluation context key for the TargetingKey field
```

Comment:
We also may want to specifically reference in some tests, to make sure we "see" breaking changes around it. I hope that makes sense.

Reply:
I've added a unit test to check for happy path plus a few other scenarios I could think of